in.varb.Ausweiskopie.yaml

```diff
@@ -0,0 +1,87 @@
+app-id: "in.varb.Ausweiskopie"
+runtime: "org.freedesktop.Platform"
+runtime-version: "24.08"
+sdk: "org.freedesktop.Sdk"
+command: ausweiskopie
+finish-args:
+  - --socket=x11
+  - --share=ipc
+modules:
+  # Step 1.1: Requirements for Tcl/Tk with proper font rendering.
+  - name: "freetype"
+    sources:
+      - type: archive
+        url: https://download.savannah.gnu.org/releases/freetype/freetype-2.13.3.tar.xz
+        sha256: 0550350666d427c74daeb85d5ac7bb353acba5f76956395995311a9c6f063289
+        x-checker-data:
+          type: html
+          url: "https://download.savannah.gnu.org/releases/freetype/"
+          version-pattern: "freetype-([\\d\\.]+\\d).tar.xz"
+          url-template: "https://download.savannah.gnu.org/releases/freetype/freetype-$version.tar.xz"
+
+  # Step 1.2: Install a current version of Tcl/Tk
+  - name: "tcl"
+    sources:
+      - type: archive
+        url: http://prdownloads.sourceforge.net/tcl/tcl8.6.16-src.tar.gz
+        sha256: 91cb8fa61771c63c262efb553059b7c7ad6757afa5857af6265e4b0bdc2a14a5
+        x-checker-data:
+          type: html
+          url: "https://sourceforge.net/projects/tcl/rss"
+          version-pattern: "https://sourceforge.net/.+/tcl(8\\.6\\.[\\d\\.]*\\d)-src.tar.gz/download"
+          url-template: "http://prdownloads.sourceforge.net/tcl/tcl$version-src.tar.gz"
+    subdir: unix
+    post-install:
+      - "chmod +w ${FLATPAK_DEST}/lib/libtcl8.6.so"
+  - name: "tk"
+    sources:
+      - type: archive
+        url: http://prdownloads.sourceforge.net/tcl/tk8.6.16-src.tar.gz
+        sha256: be9f94d3575d4b3099d84bc3c10de8994df2d7aa405208173c709cc404a7e5fe
+        x-checker-data:
+          type: html
+          url: "https://sourceforge.net/projects/tcl/rss"
+          version-pattern: "https://sourceforge.net/.+/tk(8\\.6\\.[\\d\\.]*\\d)-src.tar.gz/download"
+          url-template: "http://prdownloads.sourceforge.net/tcl/tk$version-src.tar.gz"
+    subdir: unix
+    post-install:
+      - "chmod +w ${FLATPAK_DEST}/lib/libtk8.6.so"
+
+  # Step 2.1: Install requirements for requirements...
+  - name: "dbus"
```

Comment:
dbus is in the runtime, btw. I don't see why it is needed.

Reply:
dbus-python needs a `dbus-run-session` binary or it refuses to build